src/index.js

```diff
@@ -3,22 +3,21 @@ import ReactDOM from 'react-dom';
 import { BrowserRouter as  Route, Switch } from "react-router-dom";
```

Comment:
Add Router Here so there's not Error that occurs. 
replace 'Route' with 'Router'

Reply:
I do not understand why I need to change the Route to Router, if the about-page and landing page reflects the same syntax.

Reply:
add this
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";

Reply:
Okay, but why am I changing it from Route to Router. Please explain.

Reply:
The merge mistake on my part is the reason I was saying you can either change the "Route" import to "Router" but you can just, change the "Router" that I changed to "Route" inside index.js that would probably be easier than what I asked about before. I apologize for the confusion